app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,122 +4,102 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
-    color : $text-dark-grey;
-  }
-
-  input {
-    &:invalid,
-    &:invalid:required,
-    &:invalid:required:focus {
-      -webkit-box-shadow : none;
-      -moz-box-shadow : none;
-      box-shadow : none;
-      box-shadow : none;
-
-      border : none;
-      color : $text-dark-grey;
-    }
+    color: $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
```

Comment:
You moved that part of the code but it is still there. It would be great if you could tell me why
```css
position: absolute;
top: -9999px;
left: -9999px;
```
is better than
```css
display: none;
```